### ISSUE TITLE
what does the weight_loss mean in loss function?

### ISSUE THREAD

**Issue by ZhouJiaHuan** [nilboy/tensorflow-yolo#52]

**Comment by contactmat85** [comment]
I didn't understand it neither

**Comment by mangwaier** [comment]
weight_decay = tf.multiply(tf.nn.l2_loss(var), wd, name='weight_loss')
      tf.add_to_collection('losses', weight_decay)
As the code say, there is a weight decay based on L2.
